Felienne/hedy
# [BUG FIX] Empty code output on switching Parsons exercise

**Description**
In this PR we make a small fix where the parsons output was not correctly empties when switching exercises.

**Fixes**
This PR fixes #2918.

**How to test**
Make sure Hedy is set to English (As this is the only language with multiple exercises ATM), navigate to Parsons and make sure you finish the first exercise. When clicking "Go to next exercise" the output window should be emptied.
